urnai/sc2/actions/collectables.py

```diff
@@ -0,0 +1,142 @@
+from statistics import mean
+
+from pysc2.env import sc2_env
+
+from urnai.actions.action_space_base import ActionSpaceBase
+from urnai.sc2.actions import sc2_actions_aux as scaux
+from urnai.sc2.actions.sc2_actions import raw_functions_classes as sc2_actions
+
+
+class CollectablesActionSpace(ActionSpaceBase):
+
+    def __init__(self):
+        self.noaction = [sc2_actions["no_op"].run()]
+        self.move_number = 0
+
+        self.hor_threshold = 2
+        self.ver_threshold = 2
+
+        self.moveleft = 0
+        self.moveright = 1
+        self.moveup = 2
+        self.movedown = 3
+
+        self.excluded_actions = []
+
+        self.actions = [self.moveleft, self.moveright, self.moveup, self.movedown]
+        self.named_actions = ['move_left', 'move_right', 'move_up', 'move_down']
+        self.action_indices = range(len(self.actions))
+
+        self.pending_actions = []
+        self.named_actions = None
+
+    def is_action_done(self):
+        # return len(self.pending_actions) == 0
+        return True
+
+    def reset(self):
+        self.move_number = 0
+        self.pending_actions = []
+
+    def get_actions(self):
+        return self.action_indices
```

Comment:
I didn't understand that. Shouldn't it return the actions? Why is it returning an iterator?